src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -170,7 +170,7 @@ public function testDeprecatedSuper($class, $super, $type)
     {
         set_error_handler('var_dump', 0);
         $e = error_reporting(0);
-        trigger_error('', E_USER_DEPRECATED);
+        @trigger_error('', E_USER_DEPRECATED);
```

Comment:
Why is this even here? It doesn't have a message. Presumably the intention is to pass to the $e var to trigger_error?